### PR TITLE
Show real error message on LDAP authentification

### PR DESCRIPTION
**Pull request by rds13**
Don't force error message to just be :invalid_username !

### CODE REVIEW

File: spec/lib/portus/ldap_spec.rb [hunk 1]
```diff
@@ -304,7 +304,8 @@ def load_configuration_test
       APP_CONFIG["ldap"] = { "enabled" => true, "base" => "" }
       lm = LdapMock.new(username: "", password: "1234")
       lm.authenticate!
-      expect(lm.last_symbol).to be :invalid_username
+      expect(lm.last_symbol).to be "Password is too short (minimum is 8 characters),"\
```
Comment (flavio) [inline spec/lib/portus/ldap_spec.rb]: This should be changed to `expect(lm.last_symbol).to eq("Password is....")` to make the tests pass

**Comment by flavio** [comment]
Can you fix the broken test please?

File: lib/portus/ldap.rb [hunk 2]
```diff
@@ -35,7 +35,7 @@ def authenticate!
         # strategy.
         if @ldap.bind_as(bind_options)
           user = find_or_create_user!
-          user.valid? ? success!(user) : fail!(:invalid_username)
+          user.valid? ? success!(user) : fail!(user.errors.full_messages.join(','))
```
Comment (mssola) [inline lib/portus/ldap.rb]: I'f I'm not wrong, you can just do this without calling `join`. This is already being handled by alerts.

**Comment by Salzig** [comment]
this fixes #540

File: lib/portus/ldap.rb (hunk 2 above)
Comment (rds13) [inline lib/portus/ldap.rb]: It does display content between '[' ']'.

**Comment by mssola** [comment]
LGTM. Could you squash these commits into a single one ? Thanks!

**Comment by rds13** [comment]
@mssola this PR should be clean now.

**Comment by mssola** [comment]
@rds13 sorry for being a bit of a tease, but could you please squash your commits into one ?

**Comment by mssola** [comment]
And you also have to sign off your commit, as specified in the [contributing.md](https://github.com/SUSE/Portus/blob/master/CONTRIBUTING.md) file.

**Comment by rds13** [comment]
The commit should be signed by now.

**Comment by mssola** [comment]
Thanks :)

**Pull request merged**